doc/ITEM_SPAWN.md

```diff
@@ -117,7 +117,7 @@ The procgen_id relates directly to a `relic_procgen_data` object's id. The `rule
 "charges": [10, 100]
 ```
 
-This will create 4 items, they can have different damage levels as the damage value is rolled separately for each of these items. Each item has charges (AKA ammo) in the range of 10 to 100 (inclusive); if the item needs a magazine before it can have charges, that will be taken care of for you. Using an array (which must have 2 entries) for charges/count/damage is equivalent to writing explicit min and max values. In other words `"count": [a,b]` is the same as `"count-min": a, "count-max": b`.
+This will create 4 items; they can have different damage levels as the damage value is rolled separately for each of these items.  Each item has charges (AKA ammo) in the range of 10 to 100 (inclusive); if the item needs a magazine before it can have charges, that will be taken care of for you.  Using an array (which must have 2 entries) for charges/count/damage is equivalent to writing explicit min and max values.  In other words, `"count": [a,b]` is the same as `"count-min": a, "count-max": b`.
 
 The container is checked and the item is put inside the container, and the charges of the item are capped/increased to match the size of the container.
```

Comment:
I do not understand what "increased to match the size of the container" means here.

Reply:
This occurs as per the note about charges above
``charges`: Setting only min and not max will make the game calculate the max charges based on container or ammo/magazine capacity.``
Setting just a mininum charges value means the number of charges will conform to the container.